routes/project/location/load_balancer.rb

```diff
@@ -118,6 +123,15 @@ class Clover
         lb.incr_refresh_cert
       end
 
+      r.post "enable-cert" do
```

Comment:
Seems odd to post to `enable-cert` to disable the cert server. Maybe `modify-cert-server`? Up to you.